Anna.Lytvyn/Lecture1/index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Lesson 1 Task 1</title>
+<style>
+.center {
+    text-align: center;
+    }
+body {
+    background-color: #e6ffe6;
+}
+img {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+.centerImage
+{
+text-align:center;
+display:block;
+}
+</style>
+</head>
+<body>
+	<header>
+		<div class="center">
+			<h1>My first page</h1>
+		</div>
+	</header>
+<img src="https://images.bigcartel.com/product_images/158222326/ducklings.jpg?auto=format&fit=max&h=1000&w=1000" alt='Duckling' width=200>
+<article>
```

Comment:
Article tag here is used incorrectly.
Article tag usually wraps lot's of p, img, also can have single header and footer and multiple sections.  Basically an article itself. All that leads to better accessibility and SEO, also it's easy to read and debug afterward.
You can see it how it used in ololos.space on the main page there are a many of articles for previews, and when you open a post you have a singular article, which makes sense. It helps things like the Pocket app to properly format your article and strip out comments and other clutter.